src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts

```diff
@@ -0,0 +1,27 @@
+import { UserPointIdentifierInfo, UserEmailIdentifierInfo, UserAccountIdentifierInfo, UserAgentIdentifierInfo } from "../../../../index";
+
+export type UpdateType = 'UPSERT' | 'DELETE';
+
+export interface UserSegmentUpdateRequest {
+  feed_id: string;
+  session_id: string;
+  datamart_id: string;
+  segment_id: string;
+  user_identifiers: (UserPointIdentifierInfo | UserEmailIdentifierInfo | UserAccountIdentifierInfo | UserAgentIdentifierInfo)[];
+  ts: number;
+  type: UpdateType;
+}
+
+export interface ExternalSegmentConnectionRequest {
+  feed_id: string;
+  session_id: string;
```

Comment:
You will get a null for session_id here, I didn't set the value of the session in the connection request